src/applications/disability-benefits/526EZ/tests/itfData.js

```diff
@@ -0,0 +1,1243 @@
+// The spelling of these status types has been validated with the partner team
+
+export default {
+  id: '',
+  type: 'evss_intent_to_file_intent_to_files_responses',
+  attributes: {
+    intentToFile: [
```

Comment:
Do all these ITFs test unique scenarios? It seems many are duplicates, except for small differences in creation and expiration dates. We might be able to remove many of them without any loss in test coverage.

Reply:
This is our BE mock data, we can certainly pare this down if that's preferred.